docs/src/components/ComponentPage/AndroidComponentPage.js

```diff
@@ -29,10 +29,11 @@ import ComponentScreenshots from '../DocsPageBuilder/ComponentScreenshots';
 import ComponentPage from './ComponentPage';
 
 type Props = {
-  documentationId: string,
+  documentationId: ?string,
```

Comment:
Turns out my assumption from the last PR was wrong. There is one Android component that doesn't have an accompanying documentation link — icons.